affiliated/registry.json

```
@@ -1,5 +1,23 @@
 {
     "packages": [
+                {
+          "name": "PyAutoGalaxy",
+          "maintainer": "James Nightingale <james.w.nightingale@durham.ac.uk>",
+          "stable": true,
+          "home_url": "https://pyautogalaxy.readthedocs.io/en/latest/",
+          "repo_url": "https://github.com/Jammy2211/PyAutoGalaxy",
+          "pypi_name": "https://pypi.org/project/autogalaxy/",
+          "description": "Analysing the morphologies and structures of large multi-wavelength galaxy samples.",
```

```suggestion
          "pypi_name": "autogalaxy",
          "description": "Analysing the morphologies and structures of large multi-wavelength galaxy samples.",
          "coordinated": false,
```
Only use "project" name on PyPI; required field.